projects/1.19/assets/chalk/chalk/lang/zh_cn.json

```
@@ -0,0 +1,56 @@
+{
+  "block.chalk.chalk_mark_block": "粉笔标记",
+
+  "item.chalk.black_chalk": "黑色粉笔",
+  "item.chalk.red_chalk": "红色粉笔",
+  "item.chalk.green_chalk": "绿色粉笔",
+  "item.chalk.brown_chalk": "棕色粉笔",
+  "item.chalk.blue_chalk": "蓝色粉笔",
+  "item.chalk.purple_chalk": "紫色粉笔",
+  "item.chalk.cyan_chalk": "青色粉笔",
+  "item.chalk.light_gray_chalk": "淡灰色粉笔",
+  "item.chalk.gray_chalk": "灰色粉笔",
+  "item.chalk.pink_chalk": "粉红色粉笔",
+  "item.chalk.lime_chalk": "黄绿色粉笔",
+  "item.chalk.yellow_chalk": "黄色粉笔",
+  "item.chalk.light_blue_chalk": "淡蓝色粉笔",
+  "item.chalk.magenta_chalk": "品红色粉笔",
+  "item.chalk.orange_chalk": "橙色粉笔",
+  "item.chalk.white_chalk": "白色粉笔",
+
+  "item.chalk.chalk_box": "粉笔盒",
+  "item.chalk.chalk_box.tooltip.drawing_with": "正在使用：",
+  "item.chalk.chalk_box.tooltip.open": "<右击以打开>",
+  "container.chalk.chalk_box": "粉笔盒",
+
+  "chalk.subtitles.chalk_broken": "粉笔：碎裂",

```

```suggestion
  "chalk.subtitles.chalk_broken": "粉笔：断裂",

```
粉笔还是用”断“好一些，一搬日常交流是`粉笔断了`

---

感觉是粉笔头直接磨成粉的那种音效
不知道`粉笔：磨尽`怎么样

---

> 知道`粉笔：磨尽`怎么样

我认为不错，就这个吧